init-tools.sh

```diff
@@ -82,7 +82,7 @@ if [ "$OS" == "OSX" ]; then
   ulimit -n 2048
 fi
 
-__CLIDownloadURL=https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz
```

Comment:
I would rather copy the init-tools.sh/cmd  and Tools-Override folder from the corefx too so that we have complete match.

Reply:
They've slightly different in behavio. For example, coreclr tries to restore a CLI  based on the `uname` results directly, whereas corefx has a switch statement with fallbacks if it doesn't know about a particular RID.

If we're fine adopting that behavior in coreclr, then I see no problem bringing over the entire init-tools.* scripts from the corefx repo.

Reply:
Ah, I've missed that. Ok, let's stay on the safe side for now and keep what you have.

Reply:
Sounds good :)